share/doc/homebrew/Acceptable-Formulae.md

```diff
@@ -66,9 +66,8 @@ Formulae in the core repository must have a stable version tagged by
 the upstream project. Tarballs are preferred to git checkouts, and
 tarballs should include the version in the filename whenever possible.
 
-Software that does not provide a stable, tagged version, or had guidance to
-always install the most recent version, should be put in
-[homebrew/head-only](https://github.com/Homebrew/homebrew-head-only) or [homebrew/devel-only](https://github.com/Homebrew/homebrew-devel-only).
+Software that does not provide a stable, tagged version should be put in
```

Comment:
I'd be tempted to add a reason there as well, such as it providing terrible UX and quickly becoming outdated.

Probably won't stop many people trying to get `head-only` stuff into the core again, but gives us justification to point to in the docs if it happens.